python/configuration_manager/configurations_manager.py

```diff
@@ -3,36 +3,15 @@
 #  contributor license agreements; and to You under the Apache License,
 #  Version 2.0."
 
-from definitions import CONFIG_FILE
-from xml_parser import Parser
-from config_logger import ConfigLogger
-from directories_manager import DirectoriesManager
+# TODO: relative import instead of absolute import
+from python.configuration_manager.definitions import CONFIG_FILE
+# from definitions import CONFIG_FILE
+from python.configuration_manager.xml_parser import Parser
```

Comment:
from where do 'python' package come from?

Reply:
Its the absolute path to the module.